specification/containerregistry/resource-manager/readme.nodejs.md

```diff
@@ -7,7 +7,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 nodejs:
```

Comment:
You may consider the following solution without updating the default so that you can limit the chagne to nodejs. 

``` yaml $(nodejs)
nodejs:
  azure-arm: true
  package-name: azure-arm-containerregistry
  package-version: 2.3.0
  output-folder: $(node-sdks-folder)/lib/services/containerRegistryManagement
  generate-license-txt: true
  generate-package-json: true
  generate-readme-md: true

input-file:
- Microsoft.ContainerRegistry/stable/2017-10-01/containerregistry.json
- Microsoft.ContainerRegistry/preview/2018-02-01-preview/containerregistry_build.json  
```

Reply:
@djyou - I'm ready to sign-off and merge - let me know if you wish to push this suggested change to the nodejs config before I do so :)

Reply:
Updated as suggested. This may however introduce more maintenance confusion though. /cc @daschult

Reply:
Thanks @djyou - does this mean that you only want to publish the Nodejs SDK at this time?

Reply:
Yes, since it's a preview version, we will publish the SDK when necessary.